### ISSUE TITLE
TODO: Certificate upgrade/update path

### ISSUE THREAD

**Issue by christopherobin** [Wizcorp/Dawn#33]
Got the issue today where all the certificates expired since the default TTL is set to 2 months by vault, we need to look at either:

* Having scripts on all the machines that will renew certificates as they are about to expire
* Having scripts on all the machines that will warn the admins that certificates are about to expire

We might also want to use larger default TTLs, I've renewed all of them with 1 year TTLs for now.

**Comment by stelcheck** [comment]
@christopherobin was this completed by any chance?

**Comment by christopherobin** [comment]
No, it required modules that were added only recently in Ansible ([openssl_certificate](https://docs.ansible.com/ansible/2.4/openssl_certificate_module.html)) and I didn't have the opportunity to look at it yet.
